packages/ui/src/proposals/modals/AddNewProposal/helpers.ts

```diff
@@ -151,7 +151,10 @@ export const schemaFactory = (titleMaxLength: number, rationaleMaxLength: number
       stakingAccount: StakingAccountSchema.required('Field is required'),
     }),
     proposalDetails: Yup.object().shape({
-      title: Yup.string().required('Field is required').max(titleMaxLength, 'Title exceeds maximum length'),
+      title: Yup.string()
+        .required('Field is required')
+        .max(titleMaxLength, 'Title exceeds maximum length')
+        .matches(/^[a-zA-z 1-9]*$/, 'Title includes forbidden characters'),
```

Comment:
Don't we want at least most of ASCII ? (or maybe even a bit more)
```suggestion
        .matches(/^[\x0A-\x7E]*$/, 'Title includes forbidden characters'),
```

Reply:
As on the call, it will be updated with most ASCII chars - `\x20-\x7E`